src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Range.cs

```diff
@@ -121,12 +134,11 @@ private BoundExpression LiftRangeExpression(BoundRangeExpression node, Immutable
             Debug.Assert(condition != null);
 
             // method(left.GetValueOrDefault(), right.GetValueOrDefault())
```

Comment:
>method [](start = 15, length = 6)

nit: comment seems out-of-date.
Also, name of the local (`rangeCall`) should probably be `rangeCreation`